src/main/java/jeremy/Jeremy.java

```diff
@@ -0,0 +1,89 @@
+package jeremy;
+
+import jeremy.command.Command;
+import jeremy.exception.*;
```

Comment:
Maybe you can avoid * and list explicitly the import class as mentioned [here](https://se-education.org/guides/conventions/java/basic.html#statements)